packages/common/src/components/index.ts

```diff
@@ -1,11 +0,0 @@
-import CircularProgress from '@material-ui/core/CircularProgress';
-import Divider from '@material-ui/core/Divider';
-import Typography from '@material-ui/core/Typography';
-import ListItemText from '@material-ui/core/ListItemText';
-
-export { LoadingApp } from './LoadingApp';
-export { DataGrid } from '@material-ui/data-grid';
-export { RemoteComponent } from './RemoteComponent';
-export * from './icons';
-export * from './layout';
```

Comment:
I renamed `components` -> `ui` because I wanted a `components` directory for the floating components..